locale/en/docs/guides/backpressuring-in-streams.md

```diff
@@ -42,11 +42,10 @@ const readline = require('readline');
 // process.stdin and process.stdout are both instances of Streams
 const rl = readline.createInterface({
   input: process.stdin,
-  output: process.stdout
+  output: process.stdout,
```

Comment:
It is common in API docs and the rest code base (mostly tests?) if I recall correctly.  Let me know if I should revert.